migrations/20170508160814_app_workload_report.js

```diff
@@ -0,0 +1,16 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('workload_report', function (table) {
+    table.increments('id')
+    table.bigInteger('workload_points_id').references('id').inTable('workload_points')
+    table.timestamp('effective_from').defaultTo(knex.fn.now())
+    table.timestamp('effective_to')
+  }).catch(function (error) {

```

Comment:
WMT Data Import Design document under the ODP folder contains the remaining fields for this table.